### PR TITLE
Use tagged version of the CodeFlare operator

### PR DESCRIPTION
**Pull request by anishasthana**
The reason for this is that the 0.0.2 release for the CodeFlare operator is broken -- we had a 0.0.3 release PR merged into community operators but it hasn't come out yet. We're waiting on the catalog to be updated but community operators has had some delays (on the order of days/weeks) for updates recently.

cc @DharmitD @Jooho @MichaelClifford

### CODE REVIEW

File: tests/resources/codeflare-stack/codeflare-subscription.yaml
Comment (Jooho) [inline tests/resources/codeflare-stack/codeflare-subscription.yaml:11]: I didn't test this but I wonder if one of the operators is using Manual, and the other operators are waiting for this codeflare-operator?

**Comment by LaVLaS** [comment]
/hold

@anishasthana Can you provide a link to an issue or discussion on why this needs to be set to `Manual` approval

**Comment by anishasthana** [comment]
The reason for this is that the 0.0.2 release for the CodeFlare operator is broken -- we had a 0.0.3 release PR merged into community operators but it hasn't come out yet. We're waiting on the catalog to be updated but community operators has  had some delays (on the order of days/weeks) for updates recently.

File: tests/resources/codeflare-stack/codeflare-subscription.yaml
Comment (anishasthana) [reply tests/resources/codeflare-stack/codeflare-subscription.yaml:12]: CodeFlare v0.0.2 is the only broken version -- 0.0.1 and 0.0.3 are fine. 0.0.3 is not yet available via community operatorhub.

**Comment by Jooho** [comment]
@anishasthana This still failed with 2 components
~~~
✘ FAILURE after 1209.330s: operator-tests/odh-manifests/basictests/codeflare-stack.sh:21: executing 'oc get pod -n opendatahub |grep mcad-controller | awk '{print $2}'' expecting any result and text '1/1'; re-trying every 10s until completion or 1200.000s: the command timed out

✘ FAILURE after 1201.886s: operator-tests/odh-manifests/basictests/ray.sh:40: executing 'oc logs ray-simple-test-5bd7ccdc6d-vj9rs | grep 'Simple tests passed'' expecting any result and text 'Simple tests passed'; re-trying every 10s until completion or 1200.000s: the command timed out

/root/peak/operator-tests/odh-manifests/basictests/codeflare-stack.sh
/root/peak/operator-tests/odh-manifests/basictests/ray.sh
~~~

**Review by jbusche — APPROVED** [review]
LGTM

**Pull request merged**
